content/SCALE/SCALETutorials/SystemSettings/ManageBootEnvironSCALE.md

```diff
@@ -101,11 +101,11 @@ Click the <span class="material-icons">more_vert</span> for an inactive boot env
 
 Click **Confirm**, and then click **Activate**.
 
-The **System Boot** screen status changes to **Reboot** and the current **Active** entry changes from **Now/Reboot** to **Now**, indicating that it is the current boot environment but is not used on next boot.
-{{< /expand >}}
+The **System Boot** screen status changes to **Reboot** and the current **Active** entry changes from **Now/Reboot** to **Now**, indicating that it is the current boot environment but it is not used on the next system restart (boot operation).
+
+### Cloneing a Boot Environment
 
-{{< expand "Clone (Click to expand)" "v" >}}
-Cloning copies the selected boot environment into a new inactive boot environment that preserves the **boot-pool** state at the clone creation time.
+Cloning copies the selected boot environment into a new inactive boot environment that preserves the **boot-pool** state at the clone-creation time.
 
 Click the <span class="material-icons">more_vert</span> for a boot environment, and then select **Clone** to open the **Clone Boot Environment** window.
```

Comment:
The more_vert icon is incorrect for this version - there's a different icon used for the Clone option.